<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WPDev-Laptop-Shariful" failures="1" tests="2" name="wpdev.nx.testcases.NX_ContactFromNotification" time="104.817" errors="0" timestamp="2020-12-27T15:09:20 BDT" skipped="0">
  <testcase classname="wpdev.nx.testcases.NX_ContactFromNotification" name="nxContactFormTestCase" time="99.105">
    <failure message="The following asserts failed:
	NAME TEXT IS FAILED expected [Webdriver Automation Recently Contacted Via] but found [Webdriver Recently Contacted Via]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
NAME TEXT IS FAILED expected [Webdriver Automation Recently Contacted Via] but found [Webdriver Recently Contacted Via]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at wpdev.nx.NxContactFormNotification.contactFormNotification(NxContactFormNotification.java:69)
at wpdev.nx.NxContactFormNotification.testCaseforContactFormNotification(NxContactFormNotification.java:21)
at wpdev.nx.testcases.NX_ContactFromNotification.nxContactFormTestCase(NX_ContactFromNotification.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- nxContactFormTestCase -->
  <system-out/>
  <testcase classname="wpdev.nx.testcases.NX_ContactFromNotification" name="invokeBrowser" time="5.712"/>
  <system-out/>
</testsuite> <!-- wpdev.nx.testcases.NX_ContactFromNotification -->
